### DISCUSSION TITLE
Lazy loading grouping

### DISCUSSION THREAD

**Discussion by master8848** [TanStack/router#3061] [Q&A] [answered]
For resources it make sense to group some routes to be loaded as group.
My usage I have 2 application group on file base routing which I am unable to lazy load as a group.

**Comment by schiller-manuel** [comment]
what does this mean? please explain in detail using a minimal complete example

**Reply by master8848** [reply]
📁 App1/
   ├── 📂 Dashboard/
   │   ├── 📄 DashboardPage.js
   │   └── 📄 AnalyticsPage.js
   ├── 📂 Settings/
   │   ├── 📄 ProfilePage.js
   │   └── 📄 SecurityPage.js
   └── 📄 App1Layout.js

📁 App2/
   ├── 📂 Projects/
   │   ├── 📄 ProjectListPage.js
   │   └── 📄 ProjectDetailPage.js
   ├── 📂 Tasks/
   │   ├── 📄 TaskListPage.js
   │   └── 📄 TaskDetailPage.js
   └── 📄 App2Layout.js


app1 all children routes will get loaded together If user crosses boundry between apps in sinle router app2 routes will get loaded 

What I am using right now
=> I am using every api as lazy loaded as I have more than two application and want type safty between routes of different applications .


expection
Can declare groups of routes as lazy loaded accordingly

**Reply by master8848** [reply, marked as answer]
Reply to myself , This is now possible with mix of virtual and physical routes Not yet checked
